OpenRA.Mods.Common/Traits/Crates/Crate.cs

```diff
@@ -94,6 +95,12 @@ public Crate(ActorInitializer init, CrateInfo info)
 				SetPosition(self, init.Get<LocationInit, CPos>());
 		}
 
+		void INotifyCreated.Created(Actor self)
+		{
+			if (self.TraitOrDefault<Parachutable>() != null)
```

Comment:
Please also include the height check here, otherwise crates spawned on the ground (e.g. destroyed supply trucks) will blink the parachute opening animation.

Reply:
Fixed.